computer_vision/flip_augmentation.py

```diff
@@ -0,0 +1,119 @@
+import random
+import cv2
+import os
+import glob
+from typing import List
+
+"""
+Flip image and bounding box for computer vision task
+https://paperswithcode.com/method/randomhorizontalflip
+"""
+
+# Params
+LABEL_DIR = ''
+IMAGE_DIR = ''
+OUTPUT_DIR = ''
+FLIP_TYPE = 1  # (0 is vertical, 1 is horizontal)
+
+
+def main() -> None:
+    """
+    Get images list and annotations list from input dir.
+    Update new images and annotations.
+    Save images and annotations in output dir.
+    """
+    img_paths, annos = get_dataset(LABEL_DIR, IMAGE_DIR)
+    print('Processing...')
+    new_image, new_annos, path = update_image_and_anno(
+        img_paths, annos, FLIP_TYPE)
+
+    for index in range(len(new_image)):
+        # Get random string code: '7b7ad245cdff75241935e4dd860f3bad'
+        letter_code = random_chars(32)
+        file_name = path[index].split('/')[-1].rsplit('.', 1)[0]
+        file_root = f"{OUTPUT_DIR}/{file_name}_FLIP_{letter_code}"
+        cv2.imwrite(f"/{file_root}.jpg", new_image[index], [cv2.IMWRITE_JPEG_QUALITY, 85])
+        print(f'Success {index+1}/{len(new_image)} with {file_name}')
+        annos_list = []
+        obj = ["{} {} {} {} {}".format(*anno) for anno in new_annos[index]]
+        annos_list.append(obj[0])
+        with open(f"/{file_root}.txt", "w") as outfile:
+            outfile.write("\n".join(line for line in annos_list))
+
+
+def get_dataset(label_dir: str, img_dir: str) -> List:
```

Comment:
`from __future__ import annotations` allows us to use `list` instead of `List`.
```suggestion
def get_dataset(label_dir: str, img_dir: str) -> list:
```